themes/bootstrap3/templates/record/view.phtml

```diff
@@ -6,10 +6,16 @@
   if ($this->syndeticsPlus()->isActive()) {
     $this->headScript()->appendFile($this->syndeticsPlus()->getScript());
   }
+  // Use combined search script for versions tab results
+  if (isset($this->tabs['Versions'])) {
```

Comment:
Is there a better way to initialize tab-specific dependencies generically, rather than putting this special case here?

Reply:
One way would be to add a RecordTab/scripts/<tabname>.phtml that's included if it exists, but then that's another loop checking for mostly non-existing files. I can't really think of a better way. If you have an idea, I'm all ears.

Reply:
I was thinking of adding a method to the tab class that returns a list of scripts... then we could pull those in as part of the loop through the tabs. It could default to an empty array in the base class.

Reply:
I don't really like the idea of tying the scripts to the tab class either, since they're purely presentation stuff and different themes might have different needs.

Reply:
That's true, but if we can't think of a better way, it feels to me like the lesser of the available evils. I really don't like putting a versions-specific chunk of code into the generic record layout template, since that's putting very narrow, specific logic into a template that is very widely used... and it sets a precedent that invites future special cases for other things, which could snowball into a mess. I agree that tying scripts to the tab class is not ideal since the tab class shouldn't really have awareness of the theme... but maybe we can make it a little more abstract to make it more palatable, like a `hasSupportJavascript()` boolean method which triggers the theme to take some action based on convention, like load `[TabName].js` or look for a `RecordTab/[TabName]/scripts.phtml` file as you suggest. This would avoid the problem of blindly checking for the existence of tons of mostly-nonexistent files, but it would also allow us to address tab-specific scripts on a theme-by-theme basis. I realize you could still end up with a situation where one theme requires scripts and another does not, but I think that's a more theoretical than realistic scenario, and if it came up, creating a zero-byte file somewhere could probably address it well enough.

Reply:
What do you think about the last commit?

Reply:
I think it's a pretty good compromise... the biggest disadvantage is the redundancy of having to define the scripts in every context where the tab is used. Would it be better to have a global `[TabScripts]` section so scripts would only have to be defined once? I think you could probably do that pretty safely without worrying about conflicting with a record driver class name.

Reply:
Done. While I don't like mixing a magic section name with real tabs, this is admittedly much cleaner.

Reply:
Thanks, @EreMaijala! It seems there is no solution that completely avoids some compromise, but I'm reasonably comfortable with this one. I might go ahead and merge this new functionality (minus the Versions-specific configuration) to dev ahead of the rest of the PR, since the changes stand alone fairly well and it will improve the readability of this PR. I'll block out some time for merging and review next week to keep this moving forward!